KomodoPlatform/komodo-defi-framework
# fix(ARRR): store unconfirmed change output

fixes: #2273
related: https://github.com/KomodoPlatform/librustzcash/pull/10

#### To Test:
Send a transaction that includes a change output using the entire balance, then attempt to spend/create a new transaction before the initial transaction is confirmed to ensure it is properly tracked in the unspent outputs
